llvm/lib/MC/MCCASObjectV1.cpp

```diff
@@ -936,6 +937,34 @@ Error MCCASBuilder::createStringSection(
   return Error::success();
 }
 
+Error MCCASBuilder::createLineSection(
+    StringRef S, std::function<Error(StringRef)> CreateFn) {
+  BinaryStreamReader SectionReader(S, Asm.getBackend().Endian);
+  while (true) {
```

Comment:
A short comment explaining what this loop is doing might be nice. Something like:
`// Iterate over the individual .debug_line contributions in this section and pass them to CreatFn.`
and then:
`// Extract the length of this .debug_line contribution from its header.`